mypyc/test-data/run.test

```diff
@@ -1035,14 +1035,17 @@ def u(x: int) -> int:
     d.update(x=x)
     return d['x']
 
+def get_content(d: Dict[int, int]) -> Tuple[List[int], List[int], List[Tuple[int, int]]]:
+    return list(d.keys()), list(d.values()), list(d.items())
```

Comment:
Add also tests for the "plain" primitive ops that don't create a list, such as by doing `set(d.keys())`?

Might be worth testing dict subclasses as well.